### ISSUE TITLE
preflight checks on tools

### ISSUE THREAD

**Issue by nutellinoit** [sighupio/furyctl#45]
Hi team , we need a mechanism to check if all the tools required for bootstrap and cluster phase are present in the computer running furyctl.

Today we got stuck because the openvpn tool needed by furyagent was not present.

Maybe a simple `command -v target-command` can be a solution

**Comment by angelbarrera92** [comment]
> Hi team , we need a mechanism to check if all the tools required for bootstrap and cluster phase are present in the computer running furyctl.
> 
> Today we got stuck because the openvpn tool needed by furyagent was not present.
> 
> Maybe a simple `command -v target-command` can be a solution

I did this in a `Makefile` before:

```bash
check-%: # detection of required software.
	@which ${*} > /dev/null || (echo '*** Please install `${*}` ***' && exit 1)
```

**Comment by ralgozino** [comment]
is this a duplicate of #30?

**Comment by nutellinoit** [comment]
I don't think so, that one is for logical preflight, this issue is about cli tools requirements already present on the machine that runs furyctl
